tests/unit/task_test.py

```diff
@@ -387,32 +388,32 @@ def square_minus_one():
 
     with context1:
         r1 = square._result
-        assert len(square._result._storage.listeners2["t1"]) == 0
+        assert len(get_storage(square._result).listeners2["t1"]) == 0
         square(5)
         assert square._last_finished_event  # type: ignore
         square._last_finished_event.wait()  # type: ignore
         # accessing will add it to the listeners
-        assert len(square._result._storage.listeners2["t1"]) == 0
+        assert len(get_storage(square._result).listeners2["t1"]) == 0
         assert square_minus_one.value == 24
         assert called == 1
-        assert len(square._result._storage.listeners2["t1"]) == 1
+        assert len(get_storage(square._result).listeners2["t1"]) == 1
         # assert square_minus_one._auto_subscriber.value.reactive_used == {square.value}
 
     with context2:
         r2 = square._result
-        assert len(square._result._storage.listeners2["t2"]) == 0
+        assert len(get_storage(square._result).listeners2["t2"]) == 0
         square(6)
         assert square._last_finished_event  # type: ignore
         square._last_finished_event.wait()  # type: ignore
-        assert len(square._result._storage.listeners2["t2"]) == 0
+        assert len(get_storage(square._result).listeners2["t2"]) == 0
         assert square_minus_one.value == 35
         assert called == 2
-        assert len(square._result._storage.listeners2["t2"]) == 1
+        assert len(get_storage(square._result).listeners2["t2"]) == 1
         # square_minus_one._auto_subscriber.value.reactive_used == {square.value}
 
     with context1:
         assert r1 is square._result
-        # assert len(square.result._storage.listeners2["t1"]) == 1
+        # assert len(square.result.listeners2["t1"]) == 1
```

Comment:
I'm guessing this can be either removed, or adapted to use `get_storage(...)`?

```suggestion
```

Reply:
I was doing a replace-all, and not sure what I intend here, so i left it here fore future us to take a look at.